examples/README.md

```diff
@@ -665,6 +666,46 @@ python /var/tmp/qsim-example.py
 [cqsdk]: https://developer.nvidia.com/cuquantum-sdk
 [cudatk]: https://developer.nvidia.com/cuda-toolkit
 
+### [google-cloud-storage.yaml] ![community-badge] ![experimental-badge]
+
+[google-cloud-storage.yaml]: ../community/examples/google-cloud-storage.yaml
+
+This example demonstrates several different ways to use Google Cloud Storage
+(GCS) buckets in the HPC Toolkit. There are two buckets referenced in the
+example:
+
+1. A GCS bucket that is created by the HPC Toolkit (`id: new-bucket`).
+1. A GCS bucket that is created externally from the HPC Toolkit but referenced
+   by the blueprint (`id: existing-bucket`).
+
+The created VM (`id: workstation`) references these GCS buckets with the `use`
+field. On VM startup gcsfuse will be installed, if not already on the image, and
+both buckets will be mounted under the directory specified by the `local_mount`
+option.
+
+The `wait-for-startup` module (`id: wait`) makes sure that terraform does not
+exit before the buckets have been mounted.
+
+To use the blueprint you must supply the project id and the name of an existing
+bucket:
+
+```shell
+./ghpc create community/examples/google-cloud-storage.yaml \
+  --vars project_id=<project_id> \
+  --vars existing_bucket_name=<name_of_existing_bucket>
+```
+
```

Comment:
Isn't it standard for examples to show how much quota they require?

Reply:
It is not standard. None of the other community experimental examples have quota statements. Also this example only requires 2 x vCPU of e2 which I would bet is included in every starter project. I would propose that it is not worth the risk of drift to include it in this case.